SplunkAppForWazuh/appserver/static/js/config/routes/agents-states.js

```diff
@@ -312,6 +312,25 @@ define(['../module'], function (module) {
         }]
       }
     })
+
+    // agents - CIS-CAT
```

Comment:
It'd be necessary to change this resolve to the new structure within a try catch block.